cross-project-tests/lit.cfg.py

```diff
@@ -144,32 +142,13 @@ def configure_dexter_substitutions():
         dependencies = ["clang", "lldb"]
         dexter_regression_test_c_builder = "clang"
         dexter_regression_test_cxx_builder = "clang++"
-        dexter_regression_test_debugger = "lldb"
+        dexter_regression_test_debugger = "lldb-dap"
+        dexter_regression_test_additional_flags = f'--lldb-executable "{lldb_dap_path}"'
         dexter_regression_test_c_flags = "-O0 -glldb -std=gnu11"
         dexter_regression_test_cxx_flags = "-O0 -glldb -std=gnu++11"
-        dexter_regression_test_additional_flags = '--lldb-executable "{}"'.format(
-            lldb_path
-        )
 
     tools.append(
-        ToolSubst("%dexter_regression_test_c_builder", dexter_regression_test_c_builder)
-    )
-    tools.append(
-        ToolSubst(
-            "%dexter_regression_test_cxx_builder", dexter_regression_test_cxx_builder
-        )
-    )
-    tools.append(
-        ToolSubst("%dexter_regression_test_debugger", dexter_regression_test_debugger)
-    )
-    # We don't need to distinguish cflags and ldflags because for Dexter
-    # regression tests we use clang to drive the linker, and so all flags will be
-    # passed in a single command.
-    tools.append(
-        ToolSubst("%dexter_regression_test_c_flags", dexter_regression_test_c_flags)
-    )
-    tools.append(
-        ToolSubst("%dexter_regression_test_cxx_flags", dexter_regression_test_cxx_flags)
```

Comment:
Why remove these substitutions, are they unused? If so do we still need the variables e.g. `dexter_regression_test_c_builder`